src/core/integrate.cpp

```diff
@@ -25,6 +25,28 @@
  *  see \ref integrate.hpp "integrate.hpp".
 */
 
+/*
+#########################################
+Fragen:
+-rescale vel/force: warum rescaling?
+-bei calc_forces: warum jedes mal erneut init_forces? 
+-warum inline fkt für inline force???
+-wo werden forces hingeschrieben?
+-warum bond forces bei single particle force?
```

Comment:
I agree that the name is misleading here, what it is supposed to mean is that it does not act on particle interaction pairs like the short-range IAs (e.g. Lennard-Jones).